opentelemetry-sdk/src/opentelemetry/sdk/metrics/export/__init__.py

```diff
@@ -61,14 +74,27 @@ def export(
             The result of the export
         """
 
+    @abstractmethod
     def shutdown(self) -> None:
         """Shuts down the exporter.
 
         Called when the SDK is shut down.
         """
 
 
-class ConsoleMetricsExporter(MetricsExporter):
+class CumulativeMetricsExporter(MetricsExporter):
```

Comment:
Why/what are these changes for? It seems like these are unrelated to the initial issue?